cmd/build/helmify/kustomize-for-helm.yaml

```diff
@@ -91,6 +91,7 @@ spec:
             - HELMSUBST_DEPLOYMENT_CONTROLLER_MANAGER_DISABLED_BUILTIN
             - HELMSUBST_DEPLOYMENT_CONTROLLER_MANAGER_EXEMPT_NAMESPACES
             - HELMSUBST_DEPLOYMENT_CONTROLLER_MANAGER_EXEMPT_NAMESPACE_PREFIXES
+            - HELMSUBST_DEPLOYMENT_CONTROLLER_MANAGER_EXTRA_ARGS
```

Comment:
Thanks for the PR @gracedo!
In this case, I think it's more readable and debuggable if we explicitly have a helm values field to set the `tls-min-version` flag instead of a generic arg field.

Reply:
@ritazh Thanks for reviewing! I will add a field for `tls-min-version` - but I think keeping the `extraArgs` might still be useful in general and keep things flexible, wdyt? Otherwise, maybe we should explicitly add an option for all the possible flags that can be passed into the controller manager?

Reply:
so far we have been explicitly creating fields for each flag so that by enabling a field, you are enabling the feature and its related resources can all be conditionally added/removed.

Reply:
Reverted the generic args in https://github.com/open-policy-agent/gatekeeper/pull/2289/commits/80e7d30b47e168eab8b91c187ebacf7ec8b57f66 and added new `tlsMinVersion` field in https://github.com/open-policy-agent/gatekeeper/pull/2289/commits/3d2446efda4b00684a87249585a16914b618d8d1